nservicebus/azure-service-bus/performance-tuning_body_asb_[,7).partial.md

```diff
@@ -0,0 +1,14 @@
+### CPU vs IO bound processing
+
+The following settings are used in order to tune performance in Azure Service Bus version 6 and below:
+
+ - `MaximumConcurrencyLevel`
+ - `BatchSize`
+ - `LockDuration`
+ - `MaxDeliveryCount`
+
+In scenarios where handlers are CPU intense and have very little IO, it is advised to lower the number of threads to one and increase the `BatchSize`. `LockDuration` and `MaxDeliveryCount` might require an adjustment to match the batch size, taking into account the number of messages that end up in the dead letter queue.
```

Comment:
> it is advised to lower the number of threads

We don't have threads, only maximum concurrency. IO bound benefits from allowing more concurrency at the trad e of or increasing latency. Increasing concurrency means it's likely that you also require to increase the lock duration.